pkgs/servers/dns/coredns/default.nix

```diff
@@ -2,20 +2,18 @@
 
 buildGoPackage rec {
   name = "coredns-${version}";
-  version = "005";
+  version = "010";
 
-  goPackagePath = "github.com/miekg/coredns";
+  goPackagePath = "github.com/coredns/coredns";
   subPackages = [ "." ];
 
   src = fetchFromGitHub {
-    owner = "miekg";
+    owner = "coredns";
     repo = "coredns";
     rev = "v${version}";
-    sha256 = "11wvy3xp9in5ny6h7hp24dq6asc013vrwc6bqiky83dlzasjwkf6";
+    sha256 = "1f8jclm01w6sc4f7y1lk5gg9bgkvlfdlwh1p8z0sj0v619s68xcj";
   };
 
-  goDeps = ./deps.nix;
```

Comment:
the build fails with:

```
go/src/github.com/coredns/coredns/core/dnsserver/config.go:8:2: cannot find package "github.com/mholt/caddy" in any of:
        /tmp/nix-build-coredns-010.drv-0/go/src/github.com/coredns/coredns/vendor/github.com/mholt/caddy (vendor tree)
        /nix/store/dq8ck03naq26sfkz5q1ls44fpva4b00f-go-1.8.3/share/go/src/github.com/mholt/caddy (from $GOROOT)
        /tmp/nix-build-coredns-010.drv-0/go/src/github.com/mholt/caddy (from $GOPATH)
        /nix/store/az356x1kp2gf6nglm479vkyyjvg5l0wx-govers-20150109-3b5f175/share/go/src/github.com/mholt/caddy
go/src/github.com/coredns/coredns/core/dnsserver/register.go:12:2: cannot find package "github.com/mholt/caddy/caddyfile" in any of:
        /tmp/nix-build-coredns-010.drv-0/go/src/github.com/coredns/coredns/vendor/github.com/mholt/caddy/caddyfile (vendor tree)
        /nix/store/dq8ck03naq26sfkz5q1ls44fpva4b00f-go-1.8.3/share/go/src/github.com/mholt/caddy/caddyfile (from $GOROOT)
        /tmp/nix-build-coredns-010.drv-0/go/src/github.com/mholt/caddy/caddyfile (from $GOPATH)
        /nix/store/az356x1kp2gf6nglm479vkyyjvg5l0wx-govers-20150109-3b5f175/share/go/src/github.com/mholt/caddy/caddyfile
go/src/github.com/coredns/coredns/core/zmiddleware.go:32:2: cannot find package "github.com/mholt/caddy/startupshutdown" in any of:
        /tmp/nix-build-coredns-010.drv-0/go/src/github.com/coredns/coredns/vendor/github.com/mholt/caddy/startupshutdown (vendor tree)
        /nix/store/dq8ck03naq26sfkz5q1ls44fpva4b00f-go-1.8.3/share/go/src/github.com/mholt/caddy/startupshutdown (from $GOROOT)
        /tmp/nix-build-coredns-010.drv-0/go/src/github.com/mholt/caddy/startupshutdown (from $GOPATH)
        /nix/store/az356x1kp2gf6nglm479vkyyjvg5l0wx-govers-20150109-3b5f175/share/go/src/github.com/mholt/caddy/startupshutdown
builder for ‘/nix/store/dq7g3hb5hszqgd1cmj4jll0fr93dpvnb-coredns-010.drv’ failed with exit code 1
```

Reply:
you probably want to take a look at `go2nix`.

Reply:
> the build fails...
> you probably want to take a look at go2nix.

Yeah, I know. It's still WIP. When I decided to update this package, I noticed that it's now vendoring it's dependencies so I deleted `deps.nix` and removed `goDeps` only to discover that [it's ignoring](https://github.com/coredns/coredns/blob/5eccfa2d1e4b76ac0320dc4123413cb0300a39b9/Gopkg.toml#L1) some dependencies when I tried to build. I don't have time to take care of this at the moment so I will let this sit for now.